### PR TITLE
Normalize the logging messages for easier filtering in kibana

### PR DESCRIPTION
**Pull request by Pchelolo**
The additional information has been moved to separate fields in logs,
like this we can easily filter out mesages about worker deaths and
filter a sequence of events that lead to the death of a particular
worker.

Bug: https://phabricator.wikimedia.org/T186761
cc @wikimedia/services @berndsi @mdholloway

### CODE REVIEW

File: lib/master.js [hunk 1]
```diff
@@ -175,26 +175,28 @@ class Master extends BaseService {
...
-                            // Give up.
-                            this._logger.log('fatal/service-runner/master',
-                                'startup failed, exiting master');
-                            return this._exitProcess(1);
-                        }
+                    if (!this._firstWorkerStarted
+                            && this._firstWorkerStartupAttempts++ >= STARTUP_ATTEMPTS_LIMIT) {
+                        // We tried to start the first worker 3 times, but never succeed.
+                        // Give up.
+                        this._logger.log('fatal/service-runner/master',
```
Comment (d00rman) [inline lib/master.js]: Would be good to have the exit code

**Comment by d00rman** [comment]
LGTM modulo small nit

File: lib/master.js (hunk 1 above)
Comment (Pchelolo) [reply lib/master.js]: @d00rman done

**Comment by Pchelolo** [comment]
Tagged and published as service-runner@2.5.1

**Comment by d00rman** [comment]
That was fast!

**Comment by berndsi** [comment]
@Pchelolo I think you wanted to use `msg` instead of `message`. I'm getting duplicate entries:

{"name":"service-mobileapp-node","hostname":"box","pid":6131,"level":30,"**message**":"worker shutting down","worker_pid":6131,"levelPath":"info/service-runner/worker","**msg**":"worker shutting down","time":"2018-02-08T16:36:24.717Z","v":0}

**Comment by Pchelolo** [comment]
Oh holy hell I thought we've fixed the msg/message thing. Will fix.

On the `worker_pid` thing - this particular log entry is from the worker, so yes, the worker pid will match with pid. but In case you want to filter logs to know the whole sequence of logs coming from a specific worker - including the `worker_pid` is helpful cause some of the messages will be coming from the master where the `pid` is different.

**Comment by berndsi** [comment]
Then it's ok to keep the `worker_pid`. Thanks!

**Pull request merged**